### ISSUE TITLE
where is the parameter of epoch

### ISSUE THREAD

**Issue by mingyangligithub** [zjukg/NeuralKG#27]
Dear Authors,

Thanks for your work, which provide much convenience for reproducing the KGE works. I met a problem when I reproduced the models. I found I can't change the epoch by changing the num_epoch in config. Could you give me some suggestions?

Many thanks,
Best wishes

**Comment by mingyangligithub** [comment]
The epoch doesn't depend on the setting num_epoch. Learning rate and early_stop_patience influence the total epoch.

**Comment by mingyangligithub** [comment]
I think there should be a threshold. Early_stop_patience only works when it meets the threshold. Is it correct?

**Comment by Modberge** [comment]
Thanks for your interest in our work! Actually, the maximum training epoch of NeuralKG is determined by num_epoch and early_stop_patient. In order to prevent model overfitting, we set early_stop_patience to decide whether to terminate the model early or not. If you want to train this model on a fixed epoch, you can simply set early_stop_patient to a large number (greater than num_epoch). I hope this will be of help to you ：）
